tests/kernels/quantization/test_block_fp8.py

```diff
@@ -184,7 +185,11 @@ def test_w8a8_block_fp8_matmul(M, N, K, block_size, out_dtype, seed):
 def test_w8a8_block_fp8_fused_moe(M, N, K, E, topk, block_size, dtype, seed):
     if topk > E:
         pytest.skip(f"Skipping test; topk={topk} > E={E}")
-
+    if E == 258:
+        # This block is to test out the following kernel:
+        # sgl_moe_align_block_size_fuse_share_kernel,
+        # defined at csrc/moe/moe_align_sum_kernels.cu
+        os.environ['VLLM_SHARED_EXPERT_FUSION_REPLICAS'] = "2"
```

Comment:
I'm not an expert on pytest but I think it might be better to use monkeypatch.setenv for changing env variables in tests.